docs/topics/developer-lightspeed/con_model-as-a-service.adoc

```diff
@@ -0,0 +1,23 @@
+:_newdoc-version: 2.15.0
+:_template-generated: 2024-2-21
+
+:_mod-docs-content-type: CONCEPT
+
+[id="model-as-a-service_{context}"]
+= Deploying an LLM as a scalable service 
+
+[role="_abstract"]
+
+The code suggestions differ based on various parameters about the large language model (LLM) used for an analysis. Therefore, model-as-a-service enables you more control over using {mta-dl-plugin} with an LLM that is trained for your specific requirements than general purpose models from the public AI providers.
```

Comment:
I don't think we want to be documenting model-as-a-service, IF this is implying the internal instance of https://github.com/rh-aiservices-bu/models-aas